src/GUI/Overlay/OverlaySystem.cpp

```diff
@@ -45,6 +45,12 @@ void OverlaySystem::removeOverlay(std::string_view name) {
 void OverlaySystem::drawOverlays() {
     _update();
     _renderer.drawOverlays(_nuklearContext.get());
+
+    if (_isEnabled) {
+        _renderer.beginOverlays();
+        //ImGui::ShowDemoWindow();
+        _renderer.endOverlays();
+    }
```

Comment:
If you want to give it a try you can uncomment this line and look at the demo window.